### ISSUE TITLE
Let's document: Portage

### ISSUE THREAD

**Issue by vitorhcl** [tldr-pages/tldr#12173]
### Command description

The Gentoo package manager and distribution system.

### Documentation

https://wiki.gentoo.org/wiki/Portage

### Platform

Android

### VCS repository link (e.g. GitHub, GitLab)

https://gitweb.gentoo.org/proj/portage.git

### Additional information

"For day to day usage, the emerge, emaint, and dispatch-conf will be the most used Portage commands". So I think we should document emaint and dispatch-conf first.

### Commands

- [ ] archive-conf →
- [ ] dispatch-conf →
- [x] ebuild →
- [ ] egencache →
- [ ] emaint →
- [x] emerge →
- [ ] emerge-webrsync →
- [ ] emirrordist →
- [ ] env-update →
- [ ] etc-update →
- [ ] fixpackages →
- [ ] regenworld →
- [ ] portageq →
- [ ] quickpkg →
- [ ] glsa-check →

**Comment by FII78** [comment]
Hello @vitorhcl  @kbdharun , I want to contribute to this issue by documenting the emaint and dispatch-conf commands. Could you confirm if the issue is available for contribution?

**Comment by kbdharun** [comment]
> Hello @vitorhcl @kbdharun , I want to contribute to this issue by documenting the emaint and dispatch-conf commands. Could you confirm if the issue is available for contribution?

Yes, this issue is open for contribution. Feel free to send pull requests to add the pages after reviewing [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md) and the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
